test/README.md

```diff
@@ -329,6 +332,38 @@ You also have to install its python dependencies
 pip install -r requirements.txt
 ```
 
+## BitBox02 Simulator
+
+### Dependencies
+
+In order to build the BitBox02 simulator, the following packages will need to be installed:
+
+```
+apt install docker.io
```

Comment:
Isn't there a way to build it without docker?

Reply:
Not really, this is how we build and release the BB02 firmware. Replicating the Docker environment outside of it is bound to lead to tons of overhead in maintenance and CI breaks.

podman instead of docker would also work though.

Reply:
Ok, anyway, I could build and run it. No problem.